nix/test-nixos-configuration.nix

```diff
@@ -10,6 +10,7 @@
       { from = "host"; host.port = 1337; guest.port = 1337; }
       { from = "host"; host.port = 8081; guest.port = 8081; }
       { from = "host"; host.port = 9999; guest.port = 9999; }
+      { from = "host"; host.port = 1442; guest.port = 1442; }
```

Comment:
Can you merge develop and the comment for kupo here?

Reply:
Bradley is on vacation, I'm taking over